service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationControllerTest.java

```diff
@@ -55,13 +61,29 @@
     private IdamApi idamApi;
 
     @MockBean
-    private CaseAccessApi caseAccessApi;
+    private CaseUserApi caseUserApi;
+
+    @MockBean
+    private IdamClient client;
+
+    @MockBean
+    private AuthTokenGenerator authTokenGenerator;
+
+    @BeforeEach
+    void setup() {
+        given(client.authenticateUser("fpl-system-update@mailnesia.com", "Password12")).willReturn(AUTH_TOKEN);
+
```

Comment:
these values are already defined in application yml under spring profile integration test. You already set the active profile to integration test so lets use these yml values :) 

```
  system_update:
    username: 'fpl-system-update@mailnesia.com'
    password: 'Password12'
```

Reply:
cool done :)